kubernetes/pom.xml

```diff
@@ -200,7 +200,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <swagger-core-version>1.5.12</swagger-core-version>
-    <sundrio.version>0.8.0</sundrio.version>
+    <sundrio.version>0.9.2</sundrio.version>
```

Comment:
FWIW, this also addresses an issue with efficiency when it comes to validation. +1